.github/workflows/build_ipfs.yml

```diff
@@ -0,0 +1,57 @@
+name: build_cyb_ipfs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_cyb:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.14.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build cyb for IPFS
+        run: yarn build-ipfs
+        env:
+          NODE_OPTIONS: '--max-old-space-size=8192'
+          CHAIN_ID: 'bostrom'
+
+      - name: Deploy to IPFS 👽
+        id: ipfs
+        uses: dasein108/ipfs-github-action@v1
```

Comment:
can it be organization url?

Reply:
Yes I think about to move to cybercongress repos, if I understand your tought